lib/src/database/matrix_sdk_database.dart

```diff
@@ -103,6 +103,8 @@ class MatrixSdkDatabase extends DatabaseApi with DatabaseFileStorage {
 
   late Box<String> _seenDeviceKeysBox;
 
+  late Box<Map> _userProfilesBox;
```

Comment:
do we want to add this to `importDump()` and `exportDump()`?

Reply:
I have not added those as the profiles are just cache, which can easily be loaded again from the server. Those blow up the database dump without a real benefit IMO